packages/keystone/graphiql/index.js

```diff
@@ -0,0 +1,40 @@
+const fs = require('fs')
+const path = require('path')
+
+const express = require('express')
+
+const conf = require('@open-condo/config')
+
+
+const SERVER_URL = conf['SERVER_URL'] || 'http://localhost:3000'
+
+class GraphiqlApp {
+    constructor ({
+        apiPath = '/admin/api',
+        graphiqlPath = '/admin/api',
+    } = {}) {
+        this._graphiqlHtml = this.prepareHtml(apiPath)
+        this._graphiqlPath = graphiqlPath
+    }
+
+    prepareHtml (apiPath) {
+        const fileBuffer = fs.readFileSync(path.join(__dirname, 'graphiql.html'))
+        const html = fileBuffer.toString()
+        return html
+            .replace('__REPLACE_TO_YOUR_API_PATH__', apiPath)
+            .replace('__REPLACE_TO_YOUR_SERVER_URL__', SERVER_URL)
+    }
+
+    async prepareMiddleware () {
+        // nosemrep: javascript.express.security.audit.express-check-csurf-middleware-usage.express-check-csurf-middleware-usage
+        const app = express()
+        app.get(this._graphiqlPath, (req, res) => {
```

Comment:
may be we need to move it to another URL?

Reply:
We can try to leave the old playground, but move it to another URL. 
And leave the new playground on the current URL, so as not to update the links in all the documentation.

The old playground is almost impossible to use due to overlapping popups when writing queries, but there you can change URL to API

Reply:
moved the new playground to another URL